scripts/kind-e2e/e2e.sh

```diff
@@ -80,7 +80,7 @@ function setup_broker() {
          sd=
          [[ $lighthouse = true ]] && sd=--service-discovery
          gn=
-         [[ $globalnet = true ]] && gn=--globalnet-enable
+         [[ $globalnet = true ]] && gn=--globalnet
```

Comment:
I thought to be consistent with "service-discovery" and call it "global-net" (with a dash). What do you think ?

Reply:
`Globalnet` is a the feature name unlike `Service Discovery`. We use Globalnet everywhere, not Global Net.